saleor/graphql/page/types.py

```diff
@@ -61,7 +61,9 @@ def resolve_page_type(root: models.Page, info):
 
     @staticmethod
     def resolve_content_json(root: models.Page, info):
-        return root.content
+        content = root.content
+
```

Comment:
Can we remove extra empty lines in resolvers? I think they are not necessary since these functions are short and simple.